journal-of-the-south-african-veterinary-association.csl

```diff
@@ -0,0 +1,620 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Journal of the South African Veterinary Association</title>
+    <title-short>JSAVA</title-short>
+    <id>http://www.zotero.org/styles/journal-of-the-south-african-veterinary-association</id>
+    <link href="http://www.zotero.org/styles/journal-of-the-south-african-veterinary-association" rel="self"/>
+    <link rel="template" href="http://www.zotero.org/styles/apa"/>
+    <link href="http://openjournals.net/files/Ref/HARVARD2009%20Reference%20guide.pdf" rel="documentation"/>
```

Comment:
Did you rely on the documentation at this link? That PDF looks to be from 2009.

I looked at http://www.jsava.co.za/index.php/jsava/pages/view/authors, and it has a link to http://aosis.co.za/documents/Harvard_Reference_Style_Guide.pdf (see "The Journal of the South African Veterinary Association uses the Harvard referencing style."), last updated in 2016.